### DISCUSSION TITLE
Marp featured on VSCode Marketplace

### DISCUSSION THREAD

**Discussion by noraj** [marp-team/marp#154] [Show and tell]
![image](https://user-images.githubusercontent.com/16578570/130785261-8011c077-60ac-4e86-b477-3f56d6655262.png)

**Comment by yhatt** [comment]
Marp had featured from this August for 2 months! :smile: Well, what happened to the trend about Marp extension?

![acquisition-trend](https://user-images.githubusercontent.com/3993388/135633195-c0dea782-261c-40f6-96ab-f0b09b82f2c1.png)

Earned over 4 times page views than usual, and increased the count of installation from the marketplace page in response to that :star_struck:
And there is slightly growth about installation from VS Code. 😋

**Comment by yhatt** [comment]
Now Marp extension featured again! :smile:

> ![](https://github.com/user-attachments/assets/614a5bf9-d68c-4040-b55a-f07fa9cc3cbe)
>
> _&mdash; https://github.com/microsoft/vscode/issues/141472#issuecomment-2797471069_
